packages/babel-plugin-transform-react-jsx/src/create-plugin.ts

```diff
@@ -149,7 +149,7 @@ You can set \`throwIfNamespace: false\` to bypass this warning.`,
             let pragmaFragSet = !!options.pragmaFrag;
 
             if (file.ast.comments) {
-              for (const comment of (file.ast.comments: Array<Object>)) {
+              for (const comment of file.ast.comments as Array<any>) {
```

Comment:
I wonder if we can just remove this type annotation. It was needed to produce a smaller output when we compiled `for-of` on our own sources, but it's not needed anymroe.

Reply:
let me try - I guess it should be safe to remove